xml/net_bonding.xml

```diff
@@ -56,6 +56,7 @@
     <para>
      Traffic is split between all available interfaces based on the following
      policy: <literal>[(source MAC address XOR'd with destination MAC address
+      <!-- 2022-04-01 tbazant: how to update this one? -->
```

Comment:
I'm not really sure myself - I didn't change it in the Smart Doc either :confused:

Reply:
in some comments in the code i replaced something similar by "number of devices included in the bonding" ... Far from perfect, too long ... but only i could imagine.